src/views/layouts/profile.hbs

```diff
@@ -4,6 +4,9 @@
    <body>
       {{{body}}}
       {{> footer}}
+      {{#each js}}
+      <script type="text/javascript" src="/js/{{this}}.js"></script>
+      {{/each}}
```

Comment:
@Angham116 since these lines are the same in every layout file, you could create a new scripts partial and move those lines into there.  Then each layout would have `{{{scripts}}}` near the bottom.

Reply:
@tomduggan85 done.

Reply:
@Angham116 Nice work! thanks for doing that!